### DISCUSSION TITLE
How to use multiple values for setlocal?

### DISCUSSION THREAD

**Discussion by og900aero** [gokcehan/lf#1442] [General]
I want to use setlocal command with multiple value: time and reverse
I try:
setlocal /home/Data/Kepek/ sortby time, reverse
but nothing happend.

**Comment by joelim-work** [comment]
You can't combine multiple settings for `setlocal`, they must be declared separately:

```
setlocal /home/Data/Kepek/ sortby time
setlocal /home/Data/Kepek/ reverse
```

**Comment by og900aero** [comment]
> You can't combine multiple settings for `setlocal`, they must be declared separately:
> 
> ```
> setlocal /home/Data/Kepek/ sortby time
> setlocal /home/Data/Kepek/ reverse
> ```
Thx, working great.
